modules/ocf/facts.d/dev-config

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -eu
+if hostname | grep -qE '^dev-'; then
```

Comment:
Not important but I'm pretty sure the `-E` doesn't do anything here, unless you know somewhere we don't use GNU grep.

Reply:
ocf-nfs also uses grep -qE, so I thought it best to follow convention.